### PR TITLE
docs: serial number

### PR DESCRIPTION
**Pull request by akshayitzme**
- [x] Docs: Serial Number

### CODE REVIEW

File: docs/.vitepress/sidebar.ts [hunk 1]
```diff
@@ -1,86 +1,87 @@
...
-        { text: 'Batches', link: '/inventory/batches' },
-        { text: 'Settings', link: '/inventory/settings' },
+        { text: "Introduction", link: "/inventory/introduction" },
+        { text: "Stock Movement", link: "/inventory/stock-movement" },
+        { text: "Shipment", link: "/inventory/shipment" },
+        { text: "Purchase Receipt", link: "/inventory/purchase-receipt" },
+        { text: "Stock Ledger", link: "/inventory/stock-ledger" },
+        { text: "Stock Balance", link: "/inventory/stock-balance" },
+        { text: "Batches", link: "/inventory/batches" },
+        { text: "Serial No", link: "/inventory/serial-no" },
```
Comment (18alantom) [inline docs/.vitepress/sidebar.ts:66]: Should be Serial Number, not Serial No.

File: docs/inventory/settings.md [hunk 2]
```diff
@@ -50,8 +50,14 @@ Checking this will allow the usage of Batches. Items will have a Has Batch
 option if the item is being tracked. It will also enable the Batch entry type
 which can be navigated to by using the Quick Search.
 
+### Enable Serial Number
+
+Checking this will allow the usage of Serial Number. Items will have a Has Serial No
+option if the item is being tracked. It will also enable the Serial No entry type
+in the Inventory tab.
```
Comment (18alantom) [inline docs/inventory/settings.md:57]: What do you mean by "Serial No entry type in the Inventory tab"?

File: docs/inventory/serial-no.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+![Mentioning Serial Numbers](./images/mentioning-serial-no.png)
+
+::: info Serial No
+If your item is serialised, you must put the Serial Number in the related
+column, one per line.
+:::
+
+## Serial No Status
+
```
Comment (18alantom) [inline docs/inventory/serial-no.md]: Not well enough formatted. See https://docs.frappebooks.com/inventory/stock-movement.html#movement-types

File: docs/inventory/serial-no.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- Serial No Status is set based on Stock Entry.
```
Comment (18alantom) [inline docs/inventory/serial-no.md]: Stock Entry is unnecessarily, capitalized. And it is vague it should be something like "stock entry eg..."

File: docs/inventory/serial-no.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+- Serial Nos can automatically be created from a Material Receipt or Purchase
+  Receipt. Serial Numbers are automatically created if you mention them in
```
Comment (18alantom) [inline docs/inventory/serial-no.md]: Tautology and vagueness. Is it the first sentence or the second one? Second sentence is redundant and untrue without any conjunction.

File: docs/inventory/serial-no.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  the Serial Nos column.
+- Only Serial Numbers with status `Active` can be delivered.
+- When a Shipment mentioning the Serial Number is created, status of
```
Comment (18alantom) [inline docs/inventory/serial-no.md]: This also happens for Material Issue. This section is incomplete.

**Review by 18alantom — CHANGES_REQUESTED** [review]

File: docs/.vitepress/sidebar.ts [hunk 7]
```diff
@@ -1,86 +1,87 @@
 export default function sidebar() {
   return [
     {
-      text: 'Basics',
+      text: "Basics",
```
[the remark this answers is not in the document]
Comment (akshayitzme) [reply docs/.vitepress/sidebar.ts]: corrected

File: docs/.vitepress/sidebar.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (akshayitzme) [reply docs/.vitepress/sidebar.ts]: corrected

File: docs/inventory/serial-no.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
+# Serial Number
+
+With a Serial Number, you can keep track of individual serialised items.
+The Serial No status will tell you its current inventory status.
```
[the remark this answers is not in the document]
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: could you elaborate?

File: docs/inventory/settings.md (hunk 2 above)
Comment (akshayitzme) [reply docs/inventory/settings.md:57]: in [#607 ](https://github.com/frappe/books/pull/605), a page was added for Serial Numbers in the Inventory section of the Sidebar.  [It is removed in](https://github.com/frappe/books/pull/605#discussion_r1183309116)  [#617](https://github.com/frappe/books/pull/617)

will remove that sentence.

File: docs/inventory/serial-no.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+## Enable Serial Number
+
+The Serial Number feature needs to be enabled first by checking the Enable Serial No.
+option under [Inventory Settings](/inventory/settings).
+
+To access the serial number list, go to: `Inventory > Serial No.`
```
[the remark this answers is not in the document]
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: corrected

File: docs/inventory/serial-no.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
+
+## Serialized Items
+
+Once you've enabled Serial No, you can mark an item as a serialised item by
+first checking `Track Item` and then `Has Serial No.`
```
[the remark this answers is not in the document]
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: corrected

File: docs/inventory/serial-no.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
...
+![Enable Has SerialNo](./images/item-has-serial-no.png)
+
+::: info Track Item
+
+Has Serial No. can be enabled only if Track Item is set.
+:::
+
+## Mentioning Item Serial Number
+
+Once a Serial Number has been enabled for an item, transactions involving the item that
```
[the remark this answers is not in the document]
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: corrected

File: docs/inventory/serial-no.md (hunk 6 above)
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: corrected

File: docs/inventory/serial-no.md (hunk 3 above)
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: corrected

File: docs/inventory/serial-no.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+
+::: info Track Item
+
+Has Serial No. can be enabled only if Track Item is set.
+:::
+
+## Mentioning Item Serial Number
+
+Once a Serial Number has been enabled for an item, transactions involving the item that 
+update inventory, such as stock movements, shipments, and purchases, must include the
```
[the remark this answers is not in the document]
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: corrected

File: docs/inventory/images/item-has-serial-no.png
[the remark this answers is not in the document]
Comment (akshayitzme) [reply docs/inventory/images/item-has-serial-no.png:1]: screenshot updated

File: docs/inventory/images/mentioning-serial-no.png
[the remark this answers is not in the document]
Comment (akshayitzme) [reply docs/inventory/images/mentioning-serial-no.png:1]: screenshot updated

File: docs/inventory/serial-no.md (hunk 6 above)
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: corrected

File: docs/inventory/serial-no.md (hunk 5 above)
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: corrected

File: docs/inventory/serial-no.md (hunk 4 above)
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: changed that line

File: docs/inventory/serial-no.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+Once a Serial Number has been enabled for an item, transactions involving the item that 
+update inventory, such as stock movements, shipments, and purchases, must include the 
+Serial No. 
+
+This can be done by clicking the edit button on the table row and selecting the Serial No.
+
+![Mentioning Serial Numbers](./images/mentioning-serial-no.png)
+
+::: info Serial No
+If your item is serialised, you must put the Serial Number in the related
```
[the remark this answers is not in the document]
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: corrected

File: docs/inventory/serial-no.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (akshayitzme) [reply docs/inventory/serial-no.md]: It's changed to,

https://github.com/frappe/books_docs/blob/954fe724b14c600ceb14dbd77bf904902aabcc2e/docs/inventory/serial-number.md?plain=1#L1-L7

[the remark this answers is not in the document]
Comment (18alantom) [reply docs/inventory/serial-no.md]: Yeah that's mostly alright, I just meant substitute **No** for **Number** everywhere.

**Comment by 18alantom** [comment]
@akshayitzme the images look better but the aspect ratio is still off (no need to fix it, I'm making the changes). Take a look:

Yours: 
item-has-serial-number

Correct:
Screen Shot 2023-05-08 at 09 22 24

When scaled to 800 (docs width), the first one is almost illegible.

**Pull request merged**